lib/models/model-api.ts

```diff
@@ -190,11 +190,20 @@ export class ModelsApi extends Construct {
                                 lisaServeEndpointUrlPs.parameterArn
                             ],
                         }),
+                        new PolicyStatement({
+                            effect: Effect.ALLOW,
+                            actions: [
+                                'secretsmanager:GetSecretValue'
+                            ],
+                            resources: ['*'],
```

Comment:
since we know the secret, can we use that to get the arn? or craft the arn instead of a * policy?

Reply:
fixed